### DISCUSSION TITLE
SRV_server's accuracy higher than SRV_server_best

### DISCUSSION THREAD

**Discussion by WeiJongYang** [NVIDIA/NVFlare#218] [Q&A] [answered]
Hi,

I encountered a small problem while doing cross validate. 
Sometimes SRV_server's accuracy higher than SRV_server_best.
Is that system test the global model and best global model in cross validation flow?
Than what causes the global model to have better results than the best global model?

**Comment by holgerroth** [comment, marked as answer]
The "best" global model is chosen based on the average validation scores from each client. This score is computed before each local training to when the client receives the current global model. This means also we can only select the best global model of the previous round using `[InTimeModelSelectionHandler](https://github.com/NVIDIA/NVFlare/blob/main/nvflare/app_common/handlers/intime_model_selection_handler.py)` after the client sends their model updates and scores of the previous global model. Therefore, it could be that your training is still converging and the final global model (SRV_server) is better than the selected best model (SRV_server_best).

**Reply by WeiJongYang** [reply]
Thanks for your detailed reply!  This make sense to me.
